File: base/src/main/java/com/mindera/skeletoid/threads/utils/ThreadUtils.java
package com.mindera.skeletoid.threads.utils;

import android.support.annotation.VisibleForTesting;

public class ThreadUtils {

    @VisibleForTesting
    ThreadUtils() {
        throw new UnsupportedOperationException();
    }

    /**
     * Gets the current thread name
     *
     * @return The current thread name in a printable string
     */
    public static String getCurrentThreadName() {
        return "[T# " + Thread.currentThread().getName() + "] ";
    }
}

File: base/src/main/java/com/mindera/skeletoid/threads/utils/ThreadUtils.kt
package com.mindera.skeletoid.threads.utils

object ThreadUtils {

    /**
     * Gets the current thread name
     *
     * @return The current thread name in a printable string
     */
    @JvmStatic
    val currentThreadName: String
        get() = "[T# " + Thread.currentThread().name + "] "

}